scripts/system/create/modules/brokenUrlReport.js

```diff
@@ -0,0 +1,390 @@
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+var brokenUrlReportHttpRequest;
+var brokenUrlReportUrlList = [];
+var brokenUrlReportInvalideUrlList = [];
+var brokenUrlReportProcessedUrlNo = 0;
+var brokenUrlReportUrlEntry;
+var brokenUrlReportMessageBox;
+var brokenUrlReportOverlayWebWindow;
+var BROKEN_URL_REPORT_YES_BUTTON = 0x4000;
+var BROKEN_URL_REPORT_NO_BUTTON = 0x10000;
+var MAX_URL_BEFORE_WARNING_FOR_LONG_PROCESS = 20;
+
+function brokenUrlReportRequestUrlValidityCheck(no) {
+    brokenUrlReportHttpRequest = new XMLHttpRequest();
+    brokenUrlReportHttpRequest.requestComplete.connect(brokenUrlReportGetResponseStatus);
+    brokenUrlReportHttpRequest.open("GET", brokenUrlReportUrlList[no].url);
+    brokenUrlReportHttpRequest.send();
+}
+
+function brokenUrlReportGetResponseStatus() {
+    if (brokenUrlReportHttpRequest.status === 0 || brokenUrlReportHttpRequest.status > 299) {
+        if (brokenUrlReportHttpRequest.status === 0) {
+            brokenUrlReportUrlList[brokenUrlReportProcessedUrlNo].validity = "0 - URL not well-formed";
+        } else {
+            brokenUrlReportUrlList[brokenUrlReportProcessedUrlNo].validity = brokenUrlReportHttpRequest.status + " - " + brokenUrlReportHttpRequest.statusText;
+        }
+        brokenUrlReportInvalideUrlList.push(brokenUrlReportUrlList[brokenUrlReportProcessedUrlNo]);
+    }
+    brokenUrlReportHttpRequest.requestComplete.disconnect(brokenUrlReportGetResponseStatus);
+    brokenUrlReportHttpRequest = null;
+    brokenUrlReportProcessedUrlNo = brokenUrlReportProcessedUrlNo + 1;
+    if (brokenUrlReportProcessedUrlNo === brokenUrlReportUrlList.length) {
+        brokenUrlReportGenerateFormatedReport(brokenUrlReportInvalideUrlList);
+        brokenUrlReportUrlList = [];
+        brokenUrlReportInvalideUrlList = [];
+        brokenUrlReportProcessedUrlNo = 0;
+    } else {
+        brokenUrlReportRequestUrlValidityCheck(brokenUrlReportProcessedUrlNo);
+    }
+}
+
+function brokenUrlReportGenerateFormatedReport(brokenUrlReportInvalideUrlList) {
+    var brokenUrlReportContent = "";
+    if (brokenUrlReportInvalideUrlList.length === 0) {
+        brokenUrlReportContent = "<h1>Broken Url Report<br><br>" + brokenUrlReportUrlList.length + " URL tested.<br><font style='color:#00FF00;'>NO ISSUE HAS BEEN FOUND.</font></h1><br><br><br>";
+        brokenUrlReportContent = brokenUrlReportContent + "<div style='width: 100%; text-align: left;'><hr><font style='font-size: 11px; font-weight: 500;'><i>This report ignores URL from the Asset Server (atp://), local drive paths or any string not starting by 'http'.</i></font></div>";
+        Script.setTimeout(function () {
+            brokenUrlReportOverlayWebWindow.emitScriptEvent(brokenUrlReportContent);
+        }, 3000);
+        return;
+    }
+    brokenUrlReportContent = "        <h1>Broken Url Report</h1>\n";
+    brokenUrlReportContent = brokenUrlReportContent + "        <table>\n";
```

Comment:
```suggestion
    brokenUrlReportContent += "        <table>\n";
```
And similarly for subsequent lines.